src/views/app/detail-panel/preview/attachments-block.jsx

```diff
@@ -39,12 +39,22 @@ function findAttachments(parts, acc) {
 	);
 }
 
-function getAttachmentsLink(messageId, messageSubject, attachments) {
+function getAttachmentsLink(messageId, messageSubject, attachments, attachmentType) {
 	if (attachments.length > 1) {
 		return `/service/home/~/?auth=co&id=${messageId}&filename=${messageSubject}&charset=UTF-8&part=${attachments.join(
 			','
 		)}&disp=a&fmt=zip`;
 	}
+	if (
+		attachmentType === 'image/png' ||
+		attachmentType === 'image/jpeg' ||
```

Comment:
a better to write this condition is
` if (includes(['image/png' ,'image/jpeg','image/jpg'],attachmentType)) {
your stuff
}`